### PR TITLE
Problem: issue #21 addressed for Nobel Prize api

### PR DESCRIPTION
**Pull request by alexandercbooth**
Solution: Added links for Nobel Prize data API in JSON format.

It's that time of year again. Here's hoping it's [Murakami's](https://en.wikipedia.org/wiki/Haruki_Murakami) year (we will know tomorrow 🎉).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ A curated list of awesome JSON datasets that don't require authentication.
 * [HTTP](#http)
 * [Media](#media)
 * [NASA](#nasa)
+* [Nobel Prize Api](#nobel-prize)
```
Comment (jdorfman) [inline README.md]: Just use: `[Nobel Prize]`

File: README.md [hunk 2]
```diff
@@ -370,6 +371,11 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
 ## Natural Disasters
 * [Earthquakes](http://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_hour.geojson)
 
+## Nobel Prize
+* [Prize](http://api.nobelprize.org/v1/prize.json?)
```
Comment (jdorfman) [inline README.md]: Remove `?` at end

File: README.md [hunk 3]
```diff
@@ -370,6 +371,11 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
...
+* [Laureate](http://api.nobelprize.org/v1/laureate.json?)
```
Comment (jdorfman) [inline README.md]: Remove `?` at end

File: README.md [hunk 4]
```diff
@@ -370,6 +371,11 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
...
+* [Country](http://api.nobelprize.org/v1/country.json?)
```
Comment (jdorfman) [inline README.md]: Remove `?` at end

File: README.md (hunk 1 above)
Comment (alexandercbooth) [inline README.md]: ✅

File: README.md (hunk 2 above)
Comment (alexandercbooth) [inline README.md]: ✅

File: README.md (hunk 3 above)
Comment (alexandercbooth) [inline README.md]: ✅

File: README.md (hunk 4 above)
Comment (alexandercbooth) [inline README.md]: ✅

**Comment by alexandercbooth** [comment]
Fixed up, thanks!

**Comment by jdorfman** [comment]
Thanks

**Pull request merged**